java/org/contikios/cooja/CoreComm.java

```diff
@@ -258,15 +258,6 @@ public static CoreComm createCoreComm(Cooja cooja, Path tempDir, String classNam
    */
   public abstract long getReferenceAddress();
 
-  /**
-   * Set the offset between absolute and relative memory addresses.
-   *
-   * @param offset Offset between absolute and relative addresses
-   */
-  public void setReferenceOffset(long offset) {
-    this.offset = offset;
```

Comment:
I think the `offset` variable in `CoreComm` is unused and should be removed after this change?

Reply:
You're right, fixed!